### PR TITLE
Update core-team.md

### PR DESCRIPTION
**Pull request by Jklawreszuk**
I think it would be worth mentioning that there are Linux platform maintainers :v:

### CODE REVIEW

File: en/contributors/core-team.md [hunk 1]
```diff
@@ -141,6 +141,8 @@ The core team is composed of members who make frequent contributions to the proj
       <td>Poland</td>
       <td>
           <ul>
+              <li>C#>
```
Comment (Eideren) [inline en/contributors/core-team.md:144]: Looks like you have a malformed tag here

Comment (Jklawreszuk) [reply en/contributors/core-team.md]: Upsy 😬 Done!

Comment (VaclavElias) [reply en/contributors/core-team.md]: Almost there!!! One more character to remove 🙂.

Like C#**>** is more than anything but we don't need to mentioned it there.

image

Comment (Jklawreszuk) [reply en/contributors/core-team.md]: Done!

**Comment by VaclavElias** [comment]
Thank you @Jklawreszuk

**Pull request merged**